types/artifacts.d.ts

```diff
@@ -466,8 +466,11 @@ declare global {
         endTime: number;
         transferSize: number;
         tag: {
-          tagName: string;
+          tagName: 'LINK'|'SCRIPT';
+          /** The value of `HTMLLinkElement.href` or `HTMLScriptElement.src`. */
           url: string;
+          /** A record of when changes to the `HTMLLinkElement.media` attribute occurred and if the new media type matched the page. */
+          mediaChanges?: Array<{href: string, media: string, msSinceHTMLEnd: number, matches: boolean}>;
```

Comment:
restored this to the artifact (implicitly there before, now part of the type). There's no audit use for it right now, but a) it's been saved in the artifact since the beginning, so we're just not _not_ keeping it around and b) it's important for smoke testing. I can't really find any other good way to test that it's being collected correctly because the end timing numbers that come from the `media` changes are too variable to reliably assert.